core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -158,7 +158,9 @@
                                    gracefulShutdownQuietPeriod, "gracefulShutdownQuietPeriod");
 
         requireNonNull(blockingTaskExecutor, "blockingTaskExecutor");
-        if (!(blockingTaskExecutor instanceof TimedExecutorService)) {
+        if (!(blockingTaskExecutor instanceof TimedScheduledExecutorService ||
+              blockingTaskExecutor instanceof
+                      io.micrometer.core.instrument.internal.TimedScheduledExecutorService)) {
```

Comment:
Could you explain why we are not using `TimedExecutorService` here?

Reply:
After rethinking, it is my mistake.  
I think The `blockingTaskExecutor` instance should be checked by all of `TimedScheduledExecutorService`, `TimedExecutorService` and `TimedExecutorService`.
Because there is no inheritance between `Timed*Executor*`s.
Please let me know if I missed something. 😀

Reply:
SGTM